### ISSUE TITLE
Open file/directory when clicking notification.

### ISSUE THREAD

**Issue by martinandrovich** [ion201/gnome-download-notify#13]
It would be very nice with an option to open the downloaded file or the directory of that file when a notification is clicked. Or an option to chose whether clicking the notification would:

- Remove notification
- Open directory
- Open file

**Comment by ion201** [comment]
Duplicate of #3 

The quicker solution is still blocked on https://bugzilla.mozilla.org/show_bug.cgi?id=1402612

I have not had time to look at the other solutions.
